typescript/infra/config/environments/mainnet3/balances/highUrgencyRelayerBalance.json

```diff
@@ -40,7 +41,7 @@
   "fluence": 500,
   "form": 0.0039,
   "forma": 1.25,
-  "fraxtal": 0.0434,
+  "fraxtal": 2.96,
```

Comment:
will update this in grafana once PR is approved, which will resolve the infra-test ci